catalog-scripts/idp-catalog-wizard-bitbucket-monorepo.py

```diff
@@ -74,7 +75,15 @@ def list_directories(workspace, username, app_password, repo_name, path, dir_pat
     print(f"Total YAML files created or updated: {yaml_files_created}")
 
 def create_or_update_catalog_info(workspace, dirName, dirPath):
-    directory = f"idp/{dirName}"
+
```

Comment:
Do we need similar handling for github?

Reply:
No, there is a difference in the responses of github and bitbuckets apis. In bitbucket we get path rather than the file name.